src/tests/Countries.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Countries from '../components/pages/Countries';
+import store from '../redux/store'; // Replace with your actual Redux store import
+
+test('renders Country component correctly', () => {
+  const { container } = render(
+    <Provider store={store}>
+      <Countries />
+    </Provider>,
+  );
+  expect(container).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly ensure that more tests particularly unit tests are created for several components of the app as they allow for verification of individual component of an application.